packages/headless/doc-parser/src/interface-resolver.test.ts

```diff
@@ -797,4 +863,52 @@ describe('#resolveInterfaceMembers', () => {
 
     expect(result).toEqual(firstExpectedMembers);
   });
+
+  it('resolves members in the correct order', () => {
+    const entryPoint = buildMockEntryPoint();
+
+    const testInterface = buildMockApiInterface({
+      name: 'HelloInterface',
+      members: [
+        mockSimpleProp('aab', 'string', true),
+        mockSimpleMethod(
```

Comment:
I would prefer to keep just one approach to creating mock data for tests ...

What if instead we extracted the sort function as a separate module, `entity-sorter.ts`?
We could then pass an array of entities directly rather than needing to use api-extractor mocks.
I think this would be much simpler and less code,